e2e/cli-e2e/tests/print-config.e2e.test.ts

```diff
@@ -6,13 +6,14 @@ const extensions = ['js', 'mjs', 'ts'] as const;
 export const configFilePath = (ext: (typeof extensions)[number]) =>
   join(process.cwd(), `e2e/cli-e2e/mocks/fixtures/code-pushup.config.${ext}`);
 
-describe('print-config', () => {
+describe('CLI print-config', () => {
   it.each(extensions)(
     'should load .%s config file with correct arguments',
     async ext => {
-      const { code, stderr, stdout } = await executeProcess({
```

Comment:
@BioPhoton here the check for `stderr` can be skipped or we could check it for just errors (ignoring warning or any other pollution)

Reply:
Thanks!